packages/IntegrationTest/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@rnw-scripts/ts-config",
+  "include": ["src"],
```

Comment:
This isn't quite right...

We're not actually type-checking outside of VS Code, since we're using Jest+Babel to run the TS directly. Can likely just remove the includes/excludes, but leave the config for VS Code.